lib.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright 2022 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+#include <unistd.h>
+
+#include <iostream>
+
+using namespace std;
+
+int GetTheAnswerToLifeTheUniverseAndEverything() {
+	if (getpid() == 1) {
+		cout << "Branch we will never hit\n";
```

Comment:
I was actually wondering while I wrote it whether this would hit if I ran it as the entrypoint of a Docker container. I think it remaps the PID to 1! 😃